numpy/core/fromnumeric.py

```diff
@@ -2230,6 +2290,13 @@ def amax(a, axis=None, out=None, keepdims=np._NoValue):
         sub-class' method does not implement `keepdims` any
         exceptions will be raised.
 
+    initializer : scalar, optional
+        The initializer for this maximum. Can also be used to compute the
+        maximum of an empty slice.
```

Comment:
Perhaps we should say `` See `~numpy.ufunc.reduce` for details`` or something to indicate that this is a simplified summary.

Reply:
Done!